guides_and_docs/computer_configuration/dotfiles_and_editors.md

```diff
@@ -3,13 +3,29 @@
 ## Dotfiles 
 [Dotfiles](https://medium.com/@webprolific/getting-started-with-dotfiles-43c3602fd789) are hidden files used to custumize your personal computer set-up. 
 
-## Emacs 
-Is a widely used and powerful editor with infinite tweeking possibilities. 
+## Editors
+A selection of suiteable modern text editors for coding.
 
+### Emacs 
+- Widely used and powerful editor with infinite tweeking possibilities.
+- Standard on most servers and HPCs
 
-## Links 
-- An [example](https://github.com/sunnivin/dotfiles) of one way one can organize dotfiles + [sunnivin's](https://github.com/sunnivin/dotfiles) personal emacs configuration.
+An [example](https://github.com/sunnivin/dotfiles) of one way one can organize dotfiles + [sunnivin's](https://github.com/sunnivin/dotfiles) personal emacs configuration.
   - Q: "I do not need emacs, I'm using vim." 
   - A: "Do you know about [emacs evil-mode](https://www.emacswiki.org/emacs/Evil)? Check it out!"
 
+### Atom
+[Webpage](https://atom.io/)
+- Beginner friendly
+- For linux
+- Large number of extensions
+- Smooth git integration
 
+### Visual Studio Code
+[Webpage](https://code.visualstudio.com/) 
+- Beginner friendly
+- Integrates well into Windows, Linux, and Mac
+- Choose from a growing number of extensions (full jupyter notebook support)
+- Work remotely through your editor
+
```

Comment:
add:
- Python file (.py), Jupyter Notebook (.ipynb), and Terminal on the same window
- Version Control Integration with Git / GitHub
- User Customization